File: lib/models/comentario_model.dart
class ComentarioModel {
  final String id;
  final String partidoId;
  final String userId;
  final String userName;
  final String texto;
  final DateTime fecha;
  final int likes;

  ComentarioModel({
    required this.id,
    required this.partidoId,
    required this.userId,
    required this.userName,
    required this.texto,
    required this.fecha,
    this.likes = 0,
  });
}

File: app/src/main/java/com/example/hockey_app/data/models/ComentarioModel.kt
package com.example.hockey_app.data.models

import kotlinx.serialization.Serializable

@Serializable
data class ComentarioModel(
    val id: String? = null,
    val partido_id: String,
    val user_id: String,
    val user_name: String,
    val texto: String,
    val fecha: String? = null,
    val likes: Int = 0
)
